src/domain/valueObject/accountId.go

```diff
@@ -10,14 +10,15 @@ import (
 type AccountId uint64
 
 func NewAccountId(value interface{}) (AccountId, error) {
-	accId, err := voHelper.InterfaceToUint64(value)
+	accountId, err := voHelper.InterfaceToUint64(value)
 	if err != nil {
-		return 0, errors.New("InvalidAccountId")
+		return 0, errors.New("AccountIdMustBeInt")
 	}
 
-	return AccountId(accId), nil
+	return AccountId(accountId), nil
 }
 
+// TODO: remove this constructor when no longer used.
```

Comment:
This _TODO_ comment will be removed once the other components that use this panic constructor are refactored and no longer rely on it.

Reply:
Right, you don't need to mention that on the code and push it to the main. You can just add to your personal to-do file or on that task to replace panics. Please, remove the todo comment from the code.

Reply:
Done.